WikipediaUnitTests/Code/UIView+VisualTestSizingUtils.m

```diff
@@ -17,6 +17,7 @@ - (CGRect)wmf_sizeThatFitsScreenWidth {
 }
 
 - (CGRect)wmf_sizeThatFitsScreenWidthForOrientation:(UIInterfaceOrientation)orientation {
+    WMF_TECH_DEBT_TODO(use nativeBounds of mainScreen)
```

Comment:
Not sure I follow - we have other visual tests that rely on the width of the device they are run on. Why does this particular suite need to be 320 pt wide?

Also if we are actually not resizing based on the screen width, then this method should be renamed - We shouldn't have to look at a method's implementation to know that it is lying to me.